build.xml

```diff
@@ -36,7 +36,7 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
 
     <property name="audience-annotations.version" value="0.5.0" />
 
-    <property name="netty.version" value="3.10.6.Final"/>
+    <property name="netty.version" value="4.1.29.Final"/>
```

Comment:
A few netty releases have happened since this code was written, consider using `4.1.33.Final` instead. It passes tests on my machine.

Reply:
I think that the upgrade is needed, but not as part of this commit.
It an be a separate change (remember to change the pom.xml file !!!)

Reply:
Sure, let's do it in a separate PR